src/form/EmailChipInput.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useCallback } from 'react';
+import { Flex, ErrorText } from 'indigo-react';
+import cn from 'classnames';
+import { useField } from 'react-final-form';
+
+import { hasErrors } from 'form/validators';
+
+const Chip = ({ value, onDelete }) => {
+  return (
+    <Flex className="r32 ph2 m1 flex-center h7 bg-gray2">
+      <Flex.Item>{value}</Flex.Item>
+      <Flex.Item onClick={onDelete} className="ml1 f6">
+        ✗
+      </Flex.Item>
+    </Flex>
+  );
+};
+
+const EmailChipInput = ({ className, name, label }) => {
+  const [chips, _setChips] = useState([]);
+
+  const [value, setValue] = useState('');
+
+  const handleChange = useCallback(
+    event => {
+      setValue(event.target.value);
+    },
+    [setValue]
+  );
+
+  const {
+    input,
+    meta: { active, error },
+  } = useField(name, {
+    subscription: { active: true, error: true },
+  });
+
+  const setChips = useCallback(
+    chips => {
+      _setChips(chips);
+      input.onChange(chips);
+    },
+    [input, _setChips]
+  );
+
+  const addToChips = useCallback(() => {
+    setChips([...chips, value]);
+    setValue('');
+  }, [chips, value, setChips]);
+
+  const handleKeyDown = useCallback(
+    event => {
+      if (['Enter', 'Tab', ','].includes(event.key) && value !== '') {
```

Comment:
Can we make space do this too? ime that's commonly supported, and it doesn't have conflicts with standard/expected behaviors like tab and return do (next field and submit respectively).

Reply:
done